Sources/ProjectDescription/ResourceFileElements.swift

```diff
@@ -7,10 +7,26 @@ import Foundation
 public struct ResourceFileElements: Codable, Equatable {
     /// List of resource file elements
     public let resources: [ResourceFileElement]
+    public let excluding: [Path]
 
     public init(resources: [ResourceFileElement]) {
         self.resources = resources
+        self.excluding = []
     }
+    
+    public init(resources: [ResourceFileElement], excluding: [Path]) throws {
+        try excluding.forEach { path in
```

Comment:
Manifests should not contain logic and therefore this should be moved to the place where the mapping logic takes place.